docs/source/visualisation/kedro-viz_visualisation.md

```diff
@@ -66,46 +66,80 @@ By convention, a [pipeline can be defined as having different layers](../resourc
 
 For example, the [data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71) labels datasets according to the stage of the pipeline (e.g. whether the data has been cleaned).
 
-You can add a `layer` attribute to the datasets in the Data Catalog, which is reflected in the Kedro-Viz visualisation.
+As of Kedro version 18.9, we've made a change to the way layers are defined in the Data Catalog. Instead of specifying the layer attribute directly within a dataset's definition, it is now included under the kedro-viz metadata. 
 
-Open `catalog.yml` for the completed spaceflights tutorial and replace the existing code with the following:
+```yaml
+companies:
+  type: pandas.CSVDataSet
+  filepath: data/01_raw/companies.csv
+  metadata: 
+    kedro-viz:
+      layer: raw
+```
+
+Before Kedro 18.9, layers were specified directly within a dataset's definition in the Data Catalog. Here's an example of the old method:
 
 ```yaml
 companies:
   type: pandas.CSVDataSet
   filepath: data/01_raw/companies.csv
   layer: raw
+```
+
+However, it is essential to be aware that this old way of defining layers will no longer be supported from Kedro version 19.0. Therefore, we highly recommend adapting to the new method where layers are specified under the kedro-viz metadata in the Data Catalog
+
+Open `catalog.yml` for the completed spaceflights tutorial and replace the existing code with the following:
```

Comment:
I think it would be better to specify this as a diff rather than new code. Will we need to update the starter too?

Reply:
i didn't find any layer reference in the starter docs. I did find in the glossary which I also updated.

Reply:
Oh, sorry, I meant we will have to update the starter code if it generates it with layers in place but I have checked now and realise that it doesn't, so no update needed. Thanks for changing the glossary.